frontend/src/components/pages/student/StudentAssessment.tsx

```diff
@@ -40,33 +41,21 @@ const AssessmentSummary = (): React.ReactElement => {
             {assessmentMetadata.startTime}
           </Text>
           <SimpleGrid columns={{ base: 1, md: 2 }} gap={14} pt="2em">
-            <Box
-              backgroundColor="rgba(232, 237, 241, 0.2)"
-              borderRadius="10px"
-              marginTop="7%"
-              padding="2em"
-            >
-              <Text color="blue.300" mb="2em" textStyle="subtitle2">
-                Assessment Question Summary
-              </Text>
-              <HStack justifyContent="space-between">
-                {/* TODO: switch to smallerParagraph after Cindy merges her PR */}
-                <Text textStyle="paragraph">Number of Questions:</Text>
-                <Text textStyle="paragraph">
-                  {assessmentMetadata.numOfQuestions}
-                </Text>
-              </HStack>
-              <HStack justifyContent="space-between">
-                <Text textStyle="paragraph">Total Number of Points:</Text>
-                <Text textStyle="paragraph">
-                  {assessmentMetadata.totalPoints}
-                </Text>
-              </HStack>
-              <br />
-              <Text textStyle="paragraph">Question Types:</Text>
-              <QuestionTypeImages
-                questionTypes={assessmentMetadata.questionTypes}
-              />
+            <Box marginTop="7%">
+              <QuestionSummary
+                questionCount={assessmentMetadata.numOfQuestions}
+                totalPoints={assessmentMetadata.totalPoints}
+              >
+                <VStack align="left" width="100%">
+                  <br />
+                  <Text paddingBottom="2" textStyle="smaller-paragraph">
+                    Question Types:
+                  </Text>
+                  <QuestionTypeImages
+                    questionTypes={assessmentMetadata.questionTypes}
+                  />
+                </VStack>
+              </QuestionSummary>
```

Comment:
could you also update line 81 to have `minWidth="10"` and remove the todo comment?